### ISSUE TITLE
Unusual case in test_load

### ISSUE THREAD

**Issue by prescod** [SFDO-Tooling/CumulusCI#1769]
David, do you remember what you were trying to test here? It seems odd to have a Lookup declaration on an Id column. (i.e. the column is both a primary key and a lookup to another table)

Am I missing something or is this a typo?

https://github.com/SFDO-Tooling/CumulusCI/blob/158a2d4356f1ac149dd72a13e961ccef8d0f9ef9/cumulusci/tasks/bulkdata/tests/test_load.py#L352

**Comment by davidmreed** [comment]
Yeah, that is a horrible hack. I _think_ the original intent was a negative test - to include a lookup that was guaranteed not to generate an after step. No idea why I called it `Id`. I'll fix it.

**Comment by prescod** [comment]
I think that there were a couple of places the pattern occurred in that file. Thanks for looking at it.

**Comment by davisagli** [comment]
@davidmreed this is still outstanding, right?

**Comment by davidmreed** [comment]
Yes, @davisagli, I need to revisit that test case.

**Comment by jstvz** [comment]
Closing as part of a backlog triage pass against v4.10.0 / 5.0.0.dev1.

Reason: stale > 24 months

If this is still reproducible against v4.10.0 (the current latest stable) or against v5.0.0.devN on the dev branch, please reopen with a fresh `cci version` block and reproduction steps. We track v5 development on the `dev` branch.
